docs/integrations/sources/github.md

```diff
@@ -209,8 +216,7 @@ The GitHub connector should not run into GitHub API limitations under normal usa
 | 0.1.6   | 2021-08-18 | [5456](https://github.com/airbytehq/airbyte/pull/5223)                                                            | Add MultipleTokenAuthenticator                                                                                                                                      |
 | 0.1.5   | 2021-08-18 | [5456](https://github.com/airbytehq/airbyte/pull/5456)                                                            | Fix set up validation                                                                                                                                               |
 | 0.1.4   | 2021-08-13 | [5136](https://github.com/airbytehq/airbyte/pull/5136)                                                            | Support syncing multiple repositories/organizations                                                                                                                 |
-| 0.1.3   | 2021-08-03 | [5156](https://github.com/airbytehq/airbyte/pull/5156)                                                            | Extended existing schemas with `users` property for certain streams                                                                                                 |
+| 0.1.3   | 2021-08-03 | [5156](https://github.com/airbytehq/airbyte/pull/5156)                                                            | Extended existing schemas with `users`property for certain streams                                                                                                  |
```

Comment:
looks like a whitespace is missing here before `property`

Reply:
Thanks!